apps/react/src/features/genres/components/GenresTextField/GenresTextField.tsx

```diff
@@ -0,0 +1,55 @@
+import { ChangeEvent, FC, memo } from 'react';
+import Box from '@mui/material/Box';
+import TextField from '@mui/material/TextField';
+import MenuItem from '@mui/material/MenuItem';
+
+/** Props. */
+type GenresTextFieldProps = Readonly<{
+
+	/** Label. */
+	label: string;
+
+	/** Query params fields. */
+	fields: Readonly<{
+
+		/** Field value. */
+		value: string;
+
+		/** Field label. */
+		label: string;
+	}>[];
+
+	/** Disabled. */
+	disabled: boolean;
+
+	/** Text field value. */
+	value: string;
+
+	/** On change text field value. */
```

Comment:
It's not a value, it's a function :)
```suggestion
	/** Callback to trigger on value changes. */
```